resources/google-music-electron-small.svg

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Hmm, maybe we should add a `README.md` to `resources` and document that for ourselves